docs/querying/sql-functions.md

```diff
@@ -2541,136 +2542,411 @@ Returns the following:
+
+```sql
+SELECT MV_CONTAINS("tags", 't3') AS contained
+FROM "mvd_example"
+LIMIT 1
+```
+
+Returns the following:
+
+| `contained` |
+| -- |
+| `true` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_FILTER_NONE
 
 Filters a multi-value expression to include no values contained in the array.
 
-* **Syntax:**: `MV_FILTER_NONE(expr, arr)`
+* **Syntax:** `MV_FILTER_NONE(expr, arr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example filters `tags` from `mvd_example` to remove values `t1` or `t3`, if present:
+
+```sql
+SELECT MV_FILTER_NONE("tags", ARRAY['t1', 't3']) AS nofilt
+FROM "mvd_example"
+LIMIT 3
+```
+
+Returns the following:
+
+| `nofilt` |
+| -- |
+| `t2` |
+| `["t4", "t5"]` |
+| `["t5","t6","t7"]` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_FILTER_ONLY
 
 Filters a multi-value expression to include only values contained in the array.
 
-* **Syntax:**: `MV_FILTER_ONLY(expr, arr)`
+* **Syntax:** `MV_FILTER_ONLY(expr, arr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example filters `tags` from `mvd_example` to only contain the values `t1` or `t3`:
+
+```sql
+SELECT MV_FILTER_ONLY("tags", ARRAY['t1', 't3']) AS filt
+FROM "mvd_example"
+LIMIT 3
+```
+
+Returns the following:
+
+| `filt` |
+| -- |
+| `["t1","t3"]` |
+| `t3` |
+| null |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_LENGTH
 
 Returns the length of an array expression.
 
-* **Syntax:**: `MV_LENGTH(arr)`
+* **Syntax:** `MV_LENGTH(arr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example returns the length of the `tags` multi-value strings from `mvd_example`:
+
+```sql
+SELECT MV_LENGTH("tags") AS len
+FROM "mvd_example"
+LIMIT 1
+```
+
+Returns the following:
+
+| `len` |
+| -- |
+| `3` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_OFFSET
 
 Returns the array element at the given zero-based index.
 
-* **Syntax:**: `MV_OFFSET(arr, long)`
+* **Syntax:** `MV_OFFSET(arr, long)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example returns the element at the third position of `tags` in `mvd_example`:
+
+```sql
+SELECT MV_OFFSET("tags", 2) AS elem
+FROM "mvd_example"
+LIMIT 1
+```
+
+Returns the following:
+
+| `elem` |
+| -- |
+| `t3` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_OFFSET_OF
 
 Returns the zero-based index of the first occurrence of a given expression in the array.
 
-* **Syntax:**: `MV_OFFSET_OF(arr, expr)`
+* **Syntax:** `MV_OFFSET_OF(arr, expr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example returns the zero-based index of the string `t3` from `tags` in `mvd_example`:
+
+```sql
+SELECT MV_OFFSET_OF("tags", 't3') AS index
+FROM "mvd_example"
+LIMIT 1
```

Comment:
IMO the example is more illustrative without the limit. In general, I don't feel like the limit is doing a whole lot in this data source that has so few rows.

Additionally, recommend including the "tags" dimension in the result for offset & ordinal seems more illustrative.